components/schemas/stacks/spec/services/loadbalancer/types/v1/routers/HttpRouterConfig.yml

```diff
@@ -14,21 +14,26 @@ properties:
     required:
```

Comment:
Since this whole struct is part of the submit, and every part is 'nullable', lets have it be both 'nullable' and 'optional'. This is a change we will need to start making across the spec, because it's technically correct. The platform will treat any nullable type as also optional on submit, but will always RETURN null in that space.

Honestly, going forward, marking everything that is nullable as also optional would probably be safest. I can't think of a situation where it's important to have the distinction.